### PR TITLE
Unit tests setup and examples

### PR DESCRIPTION
**Pull request by rbachleitner**
- Adds dependencies of jest and @vue/test-utils to package.json
- Provides some basic (more or less useful) examples of how unit tests can be written for Vue
- Unfortunately I manage to cram in some test-unrelated changes, but it shouldn't matter at this stage (relates to changes to the App.js file)

### CODE REVIEW

File: src/assets/backendTestPayload.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+export default {
+  papers: [
+    // { id: 0,
```
Comment (tarrow) [inline src/assets/backendTestPayload.js]: IMHO we should try and avoid this kind of big block of commented code in a PR. I think we should just delete and drag it out of git if we really need it.

File: src/__tests__/Paper.test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    base_unit: 'readers'
+  });
+});
+
+describe('Paper', () => {
+  it('matches a previously made snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('s g element reacts to mouseover, mouseout, click and doubleclick events', () => {
```
Comment (tarrow) [inline src/__tests__/Paper.test.js]: Cool, these looks like things that are good to test. IMHO we should split these all up in to separate tests though. I.e. this should be 4 it() blocks.

File: src/__tests__/Paper.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import Paper from '../components/Paper';
+import { shallowMount } from '@vue/test-utils';
+
+const factory = (values = {}) => {
+  return shallowMount(Paper, {
```
Comment (tarrow) [inline src/__tests__/Paper.test.js]: Cool, this is all new and snazzy eh?

File: src/__tests__/Bubble.test.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    expect(wrapper.find('g').exists()).toBe(true);
+    expect(wrapper.find('circle').exists()).toBe(true);
+    expect(wrapper.find('circle').attributes().r).toBe("30");
+    expect(wrapper.find('#bubble0').exists()).toBe(true);
+    expect(wrapper.find('#bubble0').attributes().transform).toBe("translate(100,20)");
+    expect(wrapper.find('h2').text()).toBe("Bubble Title");
+    expect(wrapper.find('foreignobject').exists()).toBe(true);
+  });
+
+  it('s circle element reacts to mouseover, mouseout, click and doubleclick events', () => {
```
Comment (tarrow) [inline src/__tests__/Bubble.test.js]: As above: I'd split these to multiple tests

File: src/assets/backendTestPayload.js (hunk 1 above)
Comment (rbachleitner) [reply src/assets/backendTestPayload.js]: Yes I missed that I can clean that up

File: src/__tests__/Paper.test.js (hunk 2 above)
Comment (rbachleitner) [reply src/__tests__/Paper.test.js]: agreed

File: src/__tests__/Paper.test.js (hunk 3 above)
Comment (rbachleitner) [reply src/__tests__/Paper.test.js]: This is pretty new I guess. But the guy wrote azorias before and this is basically the 'official' Vue azorias version.

File: src/__tests__/Bubble.test.js (hunk 4 above)
Comment (rbachleitner) [reply src/__tests__/Bubble.test.js]: agreed will do

**Comment by rbachleitner** [comment]
Cleaned up the tests and removed the changes that were not related to testing

**Pull request closed without merging**